### PR TITLE
feat(internet-header): add aria attributes to mobile menu toggle

### PR DESCRIPTION
**Pull request by alizedebray**
Added attributes recommended here: https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/internet-header/src/components/post-main-navigation/post-main-navigation.tsx [hunk 1]
```diff
@@ -250,6 +250,7 @@ export class PostMainNavigation implements HasDropdown, IsFocusable {
         <nav
           id="post-internet-header-main-navigation"
           class={{ 'main-navigation': true, 'open': this.mobileMenuOpen }}
+          role="menu"
```
Comment (oliverschuerch) [inline packages/internet-header/src/components/post-main-navigation/post-main-navigation.tsx:253]: Do you think the main navigation should have the role menu also on desktop or only on mobile devices?

**Review by oliverschuerch — COMMENTED** [review]
onFocus `button.menu-button` nvda reads the following:
IS: "Menü Menü Schaltfläche reduziert Untermenü."
SHOULD: "Hauptnavigation Menü Schaltfläche reduziert Untermenü."

File: packages/internet-header/src/components/post-main-navigation/post-main-navigation.tsx (hunk 1 above)
Comment (alizedebray) [reply packages/internet-header/src/components/post-main-navigation/post-main-navigation.tsx:253]: From what I understood of the [ARIA: menu role documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role) it seems to be appropriate for the desktop version as well. 

> A menu can be a permanently visible list of controls or a widget that can be made to open and close.

**Pull request merged**